modules/ROOT/pages/create-inbound-message-flow.adoc

```diff
@@ -1,40 +1,56 @@
 = Create and Configure an Inbound Message Flow
 :page-aliases: configure-message-flows.adoc
 
-You can create and configure an inbound message flow to receive EDI messages from your trading partners, validate and transform those messages, and then send the translated application (JSON or XML) message to your backend applications.
+You can create and configure an inbound message flow to receive EDI messages from your trading partners, including messages that go through 3rd party connections. When you create an inbound message flow, you can validate and transform the inbound messages and then send the translated message (JSON or XML) to your backend applications.
 
-== Create an Inbound Message Flow
+To receive messages sent from a trading partner through a 3rd party connection, select an endpoint owned by that connection.
 
 === Prerequisites
 Before you create your first inbound message flow, you must configure your xref:setup-payload-storage-API.adoc[payload storage API].
 
 === Steps
 
-. From your Anypoint Partner Manager sandbox or production environment, select *Message Flows* in the navigation menu.
+. From the Anypoint Partner Manager navigation menu, select *Message Flows*.
 . Click *New message flow > Receive from partner*.
-. Either select an existing partner from the list and click *Select* or select *New* to xref:configure-partner.adoc#create-partner[create a trading partner]. +
-A new message flow is created. +
+. Either select an existing partner from the list or select *New* to create a new partner, as described in xref:create-partner.adoc#[Create a partner].
++
+A new message flow is created.
```

Comment:
Is this formatted right? The "A new message flow is created."

Reply:
It works, even though I think it is better to have the + on its own line. I am changing these as I rewrite a topic.